frontend/packages/dev-console/src/components/pipelines/pipeline-builder/update-utils.ts

```diff
@@ -328,10 +328,11 @@ const updateTask: UpdateOperationAction<UpdateOperationUpdateTaskData> = (
 
   return {
     tasks: updatedTasks,
-    listTasks: listTasks.map(
-      (listTask) =>
-        canRename && mapReplaceRelatedInOthers(newName, thisPipelineTask.name, listTask),
-    ),
+    listTasks: canRename
+      ? listTasks.map((listTask) =>
+          mapReplaceRelatedInOthers(newName, thisPipelineTask.name, listTask),
+        )
+      : listTasks,
```

Comment:
Found this while playing with the resource stuff... if you update a task and it's pointing at a list node... it white screens 🤦‍♂ because of the boolean operation I was doing INSIDE the map... oi.